pennylane/measure.py

```diff
@@ -123,3 +177,48 @@ def sample(op):
         qml._current_context._append_op(op)
 
     return op
+
+
+def probs(wires):
+    r"""Probability of each computational basis state.
+
+    This measurement function accepts no observables, and instead
+    instructs the QNode to return a flat array containing the
+    probabilities of each quantum state.
```

Comment:
As an extra sentence, it might be worth adding something like the explanation in the example such as
``The dimension of the computational basis states is equal to the number of wires specified.``

Reply:
Updated to say
```rest
Marginal probabilities may also be requested by restricting
the wires to a subset of the full system; the size of the
returned array will be ``[2**len(wires)]``.
```